Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -336,11 +320,11 @@ Include individual extensions like site packages
 A project will often contain custom extensions, and at the least a :ref:`sitepackage <t3sitepackage:start>`
 that provides the TYPO3-related project templates and configuration.
 
-Before TYPO3 v12, this extensions were stored in a directory like :file:`typo3conf/ext/my_sitepackage`.
+Before TYPO3 v12, these extensions were stored in a directory like :file:`typo3conf/ext/my_sitepackage`.
 In Composer mode, you can easily add a custom repository within your project
```

Comment:
Please remove "easily". Sometimes, for users it is not easy.

Reply:
ack, rephrased.